pkg/languagedetection/caching.go

```diff
@@ -0,0 +1,5 @@
+package languagedetection
+
+type binaryId struct {
```

Comment:
This type seems to also be defined in `privileged_detector.go`

Reply:
I actually just caught that, fixed